quantgov/estimator/structures.py

```diff
@@ -5,6 +5,34 @@
 """
 import collections
 import joblib as jl
+from sklearn.base import BaseEstimator, TransformerMixin
+from six import iteritems
+from decorator import decorator
+import re
+
+try:
+    from spacy.lang.en.stop_words import STOP_WORDS
```

Comment:
If we're literally only using spacy here for the stopwords, can't we somehow find the sklearn stopwords used in the `CountVectorizer`? That's got to be importable from *somewhere*.